sxw_resource.c

```diff
@@ -150,14 +119,14 @@ void _sxw_update_resource(void) {
 	_sxw_update_root_tables(sizes);
 
 	/* Assign transpiration (resource availability) to each STEPPE functional group */
-	_transp_contribution_by_group(_resource_cur);
+	_transp_contribution_by_group(SXWResources->_resource_cur);
 
         /* Scale transpiration resources by a constant, bvt, to convert resources
          * (cm) to biomass that can be supported by those resources (g/cm) */
 	ForEachGroup(g)
 	{
 //printf("for groupName= %smresource_cur prior to multiplication: %f\n",RGroup[g]->name, _resource_cur[g]);
```

Comment:
This line can probably safely be removed.

Reply:
@fdpierson I case anyone wants to use these print statements I combined the "prior to multiplication" and "post multiplication" statements into a single statement.